awswrapper/s3.go

```diff
@@ -360,6 +422,28 @@ func (o *S3Service) ReadFromS3Concurrently(bucketName string, path string) (cont
 	return
 }
 
+func (o *S3Service) DownloadFromS3WithSession(session *ServiceSession, bucket, path string, output io.WriterAt, options *S3Options) (err error) {
```

Comment:
Same with `ReadFromS3Concurrently`

Reply:
This is like a above. We can pass a session so that we can do both upload and download in one session